test/silicon_redHF.jl

```diff
@@ -24,7 +24,8 @@ function run_silicon_redHF(T; Ecut=5, grid_size=15, spin_polarization=:none, kwa
     ]
     ref_etot = -5.440593269861395
 
-    fft_size = grid_size * ones(3)
+    fft_size = fill(grid_size, 3)
+    fft_size = DFTK.next_working_fft_size.(T, fft_size) # ad-hoc fix for buggy generic FFTs
```

Comment:
Actually I think asking the user to do this explicitly is better indeed. However in that case I would probably change the user-facing function to be just a call to a tuple and not require the extra broadcast.